kinto_elasticsearch/indexer.py

```diff
@@ -24,6 +24,13 @@ def create_index(self, bucket_id, collection_id):
         if not self.client.indices.exists(index=indexname):
             self.client.indices.create(index=indexname)
 
+    def delete_index(self, bucket_id, collection_id=None):
+        if collection_id is None:
+            collection_id = "*"
+        indexname = self.indexname(bucket_id, collection_id)
+        if self.client.indices.exists(index=indexname):
```

Comment:
Is this called multiple times? Should we plan to put them all in a ES batch? Cannot we delete an indice and fail silently if it doesn't exists to avoid making sure it exists?